wordpress-mobile/WordPress-iOS
# Unified prologue - Stats screen

Ref #15056

NOTE: this PR calculates the distance between the prologue title and the content view relative to the actual view height, and removes the `Spacer` above and below the SwiftUI view. As a consequence, if we follow this approach, other prologue pages need to be updated

To test:

- build/run and log out if needed
- Make sure the stats prologue screen look consistent with the designs (#15056). See also the example screenshots below

iPhone | iPad
-- | --
<img height=450 src=https://user-images.githubusercontent.com/34376330/112914716-5344b900-90c2-11eb-8212-3bb3830295ee.png> | <img height=450 src=https://user-images.githubusercontent.com/34376330/112914732-5fc91180-90c2-11eb-9f27-a5a18539354d.png>



PR submission checklist:
